src/app/clusters/operational-state-server/operational-state-cluster-objects.h

```diff
@@ -39,7 +41,7 @@ struct GenericOperationalState : public app::Clusters::detail::Structs::Operatio
 {
     GenericOperationalState(uint8_t state = to_underlying(OperationalStateEnum::kStopped), Optional<CharSpan> label = NullOptional)
     {
-        Set(state, label);
+        Set(state, std::move(label));
```

Comment:
These additions of std::move and the utility header are not just "moving code around".  And I'm not sure why they are here; Span has no useful move constructor.  Sunk cost, so not sure it's worth reverting (except for the "why would we do that?" part) but when a PR description says it's moves only and the actual changes are not moves only, that means all the changes have to be scrutinized carefully....

Reply:
Sorry about this. I had copied this from the other PR and forgot that I had made this change. I will revert this.

Reply:
Reverted.